<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,10">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="check_canny">
        <property name="text">
         <string>canny算子</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start_rec">
        <property name="text">
         <string>开始录制</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_stop_rec">
        <property name="text">
         <string>停止录制</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="pic_show">
      <property name="text">
       <string>显示窗口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>479</width>
         <height>358</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_file_open"/>
    <addaction name="action_dir_open"/>
    <addaction name="action_files_open"/>
    <addaction name="action_cam_open"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>算法</string>
    </property>
    <addaction name="action_meanshift"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_file_open"/>
   <addaction name="action_cam_open"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_file_open">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_dir_open">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="action_files_open">
   <property name="text">
    <string>打开多个文件</string>
   </property>
  </action>
  <action name="action_cam_open">
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </action>
  <action name="action_meanshift">
   <property name="text">
    <string>meanshift</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
